app/admin/golem/jobs/page.tsx

```
@@ -302,86 +338,437 @@ export default function JobsPage() {
+
+            {/* Filter grid - responsive */}
+            <div className="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-4 gap-4">
+              {/* Source filter */}
+              <div>
+                <label className="flex items-center gap-1.5 text-xs text-white/50 mb-2">
+                  <Building2 className="h-3 w-3" />
+                  Source
+                </label>
+                <select
+                  value={filterSource}
+                  onChange={(e) => setFilterSource(e.target.value)}
```

Biome (2.3.13)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)